eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -1,25 +1,81 @@
+[CmdletBinding(SupportsShouldProcess)]
 param(
   # The repo owner: e.g. Azure
   $RepoOwner,
   # The repo name. E.g. azure-sdk-for-java
   $RepoName,
   # Please use the RepoOwner/RepoName format: e.g. Azure/azure-sdk-for-java
   $RepoId="$RepoOwner/$RepoName",
-  [Parameter(Mandatory = $true)]
-  $BranchPrefix,  
+  # CentralRepoId the original PR to generate sync PR. E.g Azure/azure-sdk-tools for eng/common
+  $CentralRepoId,
+  # We start from the sync PRs, use the branch name to get the PR number of central repo. E.g. sync-eng/common-(<branchName>)-(<PrNumber>). Have group name on PR number.
+  $CentralPRRegex,
   # Date format: e.g. Tuesday, April 12, 2022 1:36:02 PM. Allow to use other date format.
   [AllowNull()]
   [DateTime]$LastCommitOlderThan,
+  [switch]$skipCentralPR,
```

Comment:
Do we need this? Can we not just use CentralRepoId as the check? If it doesn't exist we don't need to check a Central PR.